File: p_011_1_resvalues/src/main/java/ru/startandroid/p_011_1resvalues/MainActivity.java
package ru.startandroid.p_011_1resvalues;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);

        LinearLayout bottom11 = findViewById(R.id.bottom11);
        TextView tvBottom = findViewById(R.id.tvBottom);
        Button btnBottom = findViewById(R.id.btnBottom);

        bottom11.setBackgroundResource(R.color.bottomColor);
        tvBottom.setText(R.string.tvBottom);
        btnBottom.setText(R.string.btnBottom);
    }
}

File: k_184_notification/src/main/java/com/startandroid/k_184_notification/MainActivity.kt
package com.startandroid.k_184_notification

import android.app.Notification
import android.app.NotificationChannel
import android.app.NotificationManager
import android.os.Build
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.core.app.NotificationCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val builder: NotificationCompat.Builder = NotificationCompat.Builder(this)
            .setSmallIcon(R.drawable.baseline_heart_broken_24)
            .setContentTitle("Title")
            .setContentText("Notification text")

        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            val channelId = "my_channel_id"
            val channelName = "My Channel"
            val importance = NotificationManager.IMPORTANCE_DEFAULT
            val channel = NotificationChannel(channelId, channelName, importance)
            val notificationManager = getSystemService(NotificationManager::class.java)
            notificationManager.createNotificationChannel(channel)
            builder.setChannelId(channelId)
        }

        val notification: Notification = builder.build()

        val notificationManager: NotificationManager =
            getSystemService(NOTIFICATION_SERVICE) as NotificationManager
        notificationManager.notify(1, notification)
    }
}